packages/dataregistry/src/cachedObservable.ts

```diff
@@ -139,7 +139,7 @@ export class CachedObservable<T> extends Observable<T> {
           subscriber.next(state.value);
           return () => {};
         default:
-          const _exhaustiveCheck: never = state;
```

Comment:
Nice! This is great, since it also preserves verifying at compile time that  the check is exhaustive. Thanks.